sensor/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,119 @@
+# tag_based_pnp_calibrator
+
+A tutorial for this calibrator can be found [here](../docs/tutorials/tag_based_pnp_calibrator.md).
+
+## Purpose
+
+The package `tag_based_pnp_calibrator` allows extrinsic calibration among camera and 3d lidar sensors used in autonomous driving and robotics.
+
+## Inner-workings / Algorithms
+
+The `tag_based_pnp_calibrator` utilizes the PnP (Perspective-n-Point) algorithm, a computer vision technique that be used to estimate the transformation between a set of correspondences of 3d and 2d points (in this case the 3d points come from the lidar and the 2d points come from the camera). For the calibration process, both `apriltag_ros` and `lidartag` packages are required, and they are executed automatically by this package's launcher files.
```

Comment:
"that be used"

Reply:
@knzo25 What do you mean?

Reply:
"...  a computer vision technique that be used to estimate the transformation ..."

Reply:
@vividf you did not answered this one

Reply:
Sorry, I didn't make this commit separately before, 
I fixed it in https://github.com/tier4/CalibrationTools/pull/164/commits/f49454cead9f1321aae2e09e9901cdcbe6175489. 

 "that be used" -> "that is used"

Reply:
@vividf 
Remember to split the commits...